descope/sdk/mgmt.go

```diff
@@ -241,6 +241,14 @@ type User interface {
 	// Returns slice of users and total number of users for the query
 	SearchAll(ctx context.Context, options *descope.UserSearchOptions) ([]*descope.UserResponse, int, error)
 
+	// Search all test users according to given filters
```

Comment:
The comment is wrong here, it refers to the SearchAll above

Reply:
those two apis doing almost the same (one for regular users and one for test users only) so the comment is ok (it includes the main reason for this api which is "test" user and all other is the same)